romeo/asset-contract/contracts/asset.clar

```diff
@@ -90,5 +118,9 @@
 ;; private functions
 ;;
 (define-private (is-contract-owner)
-    (is-eq (var-get contract-owner) tx-sender)
+    (ok (asserts! (is-eq (var-get contract-owner) contract-caller) err-invalid-caller))
```

Comment:
Is this more efficient?

For a return ok, I would expect the name of the function something like `check-is-contract-owner`.

Reply:
Not sure if it is more efficient but it definitely makes using the function easier to use (with a `try!`)—and there is no chance you return the wrong error value somewhere. It is a standard pattern I follow.